gpgLabs/DC/__init__.py

```diff
@@ -1,2 +1,3 @@
-from . import DCLayers
-from . import DCWidget
+from em_examples.DCLayers import *
```

Comment:
I would recommend not importing * internally. This makes debugging a giant headache and also tends to clobber the namespace
```
from em_examples.DCLayers import class1, class2
```